beaver_cli/lib/src/command/http_command.dart

```diff
@@ -61,22 +61,10 @@ abstract class HttpCommand extends Command {
     }, help: 'Path prefix.');
   }
 
-  Uri getServerUrl() {
-    final url = new StringBuffer();
-
-    if (secure) {
-      url.write('https://');
-    } else {
-      url.write('http://');
-    }
-    url.write(address);
-    url.write(':');
-    url.write(port);
-    if (pathPrefix != null) {
-      url.write(pathPrefix);
-    }
-    url.write(api);
-
-    return Uri.parse(url.toString());
-  }
+  Uri getServerUrl() => new Uri(
+      scheme: secure ? 'https' : 'http',
+      host: address,
+      port: port,
+      path: '${pathPrefix}${api}');
```

Comment:
No, pathPrefix is `` when not specified once https://github.com/fluidic/beaver/pull/40 is landed.